distributed/scheduler.py

```diff
@@ -1033,7 +1015,7 @@ class TaskPrefix(TaskCollection):
     __slots__ = tuple(__annotations__)
 
     def __init__(self, name: str):
-        super().__init__(name)
+        TaskCollection.__init__(self, name)
```

Comment:
I think it still makes some sense to keep the baseclass for common state instantiation and getters but we should try to avoid super